ports/geographiclib/portfile.cmake

```
@@ -1,49 +1,51 @@
-vcpkg_from_sourceforge (
+vcpkg_from_sourceforge(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO geographiclib
-    REF distrib
-    FILENAME "GeographicLib-1.52.tar.gz"
-    SHA512 98a4d33764db4a4755851a7db639fd9e055dcf4f1f949258e112fce2e198076b5896fcae2c1ea36b37fe1000d28eec326636a730e70f25bc19a1610423ba6859
-    PATCHES cxx-library-only.patch
+    REF distrib-C++
+    FILENAME "GeographicLib-2.0.tar.gz"
+    SHA512 7cf67174a64082372cdd249a64460e9f61c582aaf3d2a31e4e69d811f265e078ba62f945e9f1f44be6c58de4c20d0359dd46e0fd262ffac229df0ba2c6adc848
+    )
+
+vcpkg_check_features(
+    OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES "tools" TOOLS
 )
 
-vcpkg_check_features (
-  OUT_FEATURE_OPTIONS FEATURE_OPTIONS
-  INVERTED_FEATURES
-    "tools" SKIP_TOOLS
-)
-
-if (VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)
-  set (LIB_TYPE "SHARED")
-else ()
-  set (LIB_TYPE "STATIC")
-endif ()
+if(TOOLS)
+    set(TOOL_OPTION "-DBINDIR=tools/${PORT}")
+else()
+    set(TOOL_OPTION -DBINDIR=)
+endif()
 
-vcpkg_configure_cmake (
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```

```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```
Please add double quotes to all the absolute paths.

---

done